src/S3Backup.php

```diff
@@ -120,67 +120,76 @@ public function backupTable(string $tableId, string $targetBucket, ?string $targ
             // Download all slices
             //@FIXME better temps
             $tmpFilePath = $tmp->getTmpFolder() . DIRECTORY_SEPARATOR . uniqid('sapi-export-');
-            foreach ($manifest["entries"] as $i => $part) {
-                $fileKey = substr($part["url"], strpos($part["url"], '/', 5) + 1);
+            foreach ($manifest['entries'] as $i => $part) {
+                $fileKey = substr($part['url'], strpos($part['url'], '/', 5) + 1);
                 $filePath = $tmpFilePath . '_' . md5(str_replace('/', '_', $fileKey));
                 $s3Client->getObject(array(
-                    'Bucket' => $fileInfo["s3Path"]["bucket"],
+                    'Bucket' => $fileInfo['s3Path']['bucket'],
                     'Key' => $fileKey,
                     'SaveAs' => $filePath
                 ));
                 $fh = fopen($filePath, 'r');
-                $this->s3Client->putObject([
-                    'Bucket' => $targetBucket,
-                    'Key' => $targetBasePath . str_replace('.', '/', $tableId) . '.part_' . $i . '.csv.gz',
-                    'Body' => $fh,
-                ]);
-                fclose($fh);
+                if ($fh) {
```

Comment:
tato podmienka tu predtym nebola, preco je uz potrebna?

Reply:
mam obavu, ze to moze potichu spadnut a nic nenapisat ... ak je to kvoli phpstanu ... tak by sa mala asi vyhodit exception ... kedze teraz to iba zmaze tmp subor a ziadna chyba sa nenapise .... moze sa to stat napr. ked bude plny disk alebo podobny problem .... pripadne to poriesit nejak komentarom alebo ignorovanim chyby a ponechat to bez podmienky -> spadne to v s3Client ak sa nepodari otvorit subor.

Reply:
to iste aj na ostatnych miestach

Reply:
jj je to kvůli phpstanu

Reply:
je to jen read file tak to nespadne na nedostatku místo

Reply:
ale tu exceptionu bych tam přidat mohl

Reply:
jj, ale napr. nedostatocne opravnenia a pod, a tazko by sa to debugovalo, vdaka ze si to upravil.